docs/content/components/timelineitem.md

```diff
@@ -41,15 +41,21 @@ Use `TimelineItem` to display items on a vertical timeline, connected by badge e
 | `square` | `Boolean` | `true` | Whether to round the edges of the image. |
 | `system_arguments` | `Hash` | N/A | [System arguments](/system-arguments) |
 
+:nodoc
```

Comment:
Might we omit these `nodocs` from the markdown output of the Rakefile?

Reply:
@manuelpuyol ^ thoughts?

Reply:
I resolved this problem for now by documenting the slots classes too ;)